visual_behavior/data_access/loading.py

```diff
@@ -512,6 +552,80 @@ def get_extended_stimulus_presentations_table(stimulus_presentations, licks, rew
     #         print('model outputs not available')
     return stimulus_presentations
 
+
+def get_stimulus_response_df_filepath_for_experiment(ophys_experiment_id, data_type, event_type, interpolate=True, output_sampling_rate=30):
```

Comment:
It would be helpful to have a short description of the functions. Seems like they are mostly for Marina, but it's hard to keep track of functions that get dir or file paths.